src/scenes/dashboard/newpost/index.js

```diff
@@ -262,17 +275,45 @@ class NewPost extends Component {
                                 </View>
-                                <Body style={styles.community_text_container}>
-                                    <Text style={{ color: 'white' }}>
-                                        {this.state.selectedGroupIndex == -1 ? 'Select a community' : this.state.grouplist[this.state.selectedGroupIndex].official_name}
-                                    </Text>
-                                </Body>
-                                <Right style={styles.communicty_icon_container}>
-                                    <Icon name='md-create' style={{ color: 'white' }} />
-                                </Right>
-                            </ListItem>
-                        </List>
-                        <SuggestionBox substitute={(mention) => this.substitute(mention)} displaySuggestionBox={this.state.displaySuggestionBox} userList={this.state.suggestionList} />
+                                <View style={styles.avatar_subfix} />
+                            <Body style={styles.community_text_container}>
+                                <Text style={{color: 'white'}}>
+                                    {this.state.selectedGroupIndex == -1 ? 'Select a community' : this.state.grouplist[this.state.selectedGroupIndex].official_name}
+                                </Text>
+                            </Body>
+                            <Right style={styles.communicty_icon_container}>
+                                <Icon name='md-create' style={{color: 'white'}} />
+                            </Right>
+                        </ListItem>
+                    </List>
+
+                    {
+                        this.state.displaySuggestionBox && this.state.suggestionList.length > 0
+                        ? <ScrollView style={{position: 'absolute', top: 20, zIndex: 3}}>
+                            <SuggestionBox substitute={(mention) => this.substitute(mention)} displaySuggestionBox={this.state.displaySuggestionBox} userList={this.state.suggestionList} />
+                        </ScrollView>
+                        : <ScrollView />
+                    }
+
+                    <ScrollView style={{marginTop: 0}}>
+                        <TextInput
+                            maxLength={POST_MAX_LENGTH}
+                            
+                            onSelectionChange={this.onSelectionChange}
+                            placeholderTextColor='rgba(0,0,0,0.1)'
+                            style={styles.textarea}
+                            multiline
+                            placeholder={this.placeholderTitle}
+                            value={this.state.content}
+                            onChangeText={(text) => this.changeContent(text)}
+                        />
+                    </ScrollView>
+                    <ShareFloatingAction 
+                        cb={() => this.setSelected(!this.state.share)}
```

Comment:
better to use onPress name

Reply:
ok, I will apply both the changes on next merge.